my-app/src/stylesheets/beachspeech.css

```diff
@@ -0,0 +1,35 @@
+body::-webkit-scrollbar {
+    width: 0.25rem;
+}
+
+body::-webkit-scrollbar-track {
+    background: #1e1e24;
+}
+
+body::-webkit-scrollbar-thumb {
+    background: rgba(6, 66, 115);
+}   
+
+body {
+    background-color: rgba(127, 205, 255);
+}
+
+.top-buffer {
+    margin-top: 32px;
```

Comment:
i c a lot of fixed "px" stuff, it might not be a problem rn since we have placeholders for everything but in the future this is going to be a problem. Not exactly sure how to fix this or if it even should be fixed so ima leave it up to u. 
- If you think taking this and making a scalable (mobile or pc) finalized page wont have that much of an impact you then you can leave it
- if you think it will have an impact you can try to fix it now
- or u can leave a FIXME

Im ok with anything

Reply:
Noted, I will change it so it will be percentage instead of px for better scalability.